vector/src/main/java/im/vector/app/features/home/room/detail/composer/RichTextComposerLayout.kt

```diff
@@ -188,6 +190,16 @@ internal class RichTextComposerLayout @JvmOverloads constructor(
         views.plainTextComposerEditText.addTextChangedListener(
                 TextChangeListener({ callback?.onTextChanged(it) }, { updateTextFieldBorder(isFullScreen) })
         )
+        ViewCompat.setOnReceiveContentListener(
+                views.richTextComposerEditText,
+                arrayOf("image/*"),
```

Comment:
Why not using what was used previously in ComposerEditText?
`ViewCompat.getOnReceiveContentMimeTypes(views.richTextComposerEditText) ?: arrayOf("image/*")`

Reply:
I was a bit confused by this line actually. According to the docs [`getOnReceiveContentMimeTypes()`](https://developer.android.com/reference/androidx/core/view/ViewCompat#getOnReceiveContentMimeTypes(android.view.View)) returns null be default, and otherwise returns any mime types set previously using `setOnReceiveContentListener()`.

So I thought it would be redundant in the rich text editor but didn't want to risk breaking anything by removing it from the old composer.

Reply:
I see, thanks for your investigation.